tests/extensions/extra/tables.txt

```diff
@@ -80,3 +80,13 @@ Lists are not tables
 
  - this | should | not
  - be | a | table
+
+Add tests for issue #449
+
+Odd backtics | Even backtics
```

Comment:
I can update the test, but I don't think there is any thing I can do about the commit message though.